client/src/app/site/users/components/user-detail/user-detail.component.ts

```diff
@@ -298,12 +300,10 @@ export class UserDetailComponent extends BaseModelContextComponent implements On
      * click on the delete user button
      */
     public async deleteUserButton(): Promise<void> {
-        const title = this.translate.instant('Are you sure you want to delete this participant?');
-        const content = this.user.full_name;
-        if (await this.promptService.open(title, content)) {
-            await this.repo.delete(this.user);
+        try {
+            await this.memberService.delete([this.user]);
```

Comment:
although the error was thrown, after "not" deleting a user the next line was evaluated. Meaning: we would come back into the user list after aborting the delete action, this was surely unexpected. Better to evaluate if the delete request actually returns something and then decide to navigate or stay

Reply:
Yeah, I do not like this implementation any more than you do, but I do not like it to return something to continue through the program flow. But, in this case returning a boolean is very likely the best option.